src/components/FormInput/FormInput.js

```diff
@@ -30,12 +30,23 @@ const FormInput = ({ name, label, children, icon: Icon, type = 'text', component
   );
 };
 
+FormInput.defaultProps = {
```

Comment:
Leave the defaultProps in the previous version. ES6 default parameters are perfectly fine.